mpisppy/utils/xhat_eval.py

```diff
@@ -57,7 +57,8 @@ def __init__(
         #TODO: CHANGE THIS AFTER UPDATE
         self.PH_extensions = None
         
-    
+        self.subproblem_creation(self.verbose)
+        self._create_solvers()
```

Comment:
The classes derived from `Xhat_Eval` probably also do this somewhere. We should eliminate the associated calls in the derived classes to ensure we're not doing extra work.